core/spec.md

```diff
@@ -4017,6 +4045,10 @@ and the following Resource level attributes:
   rules defined by the current value of this attribute. This includes rejecting
   requests to add/delete/modify Versions or update this attribute in such a
   way as to make the existing Versions incompatible due to the new value.
+  For `compatibility` strategies that require understanding the sequence in
+  which versions need to be compatible, the server MUST use the [`ancestor`]
+  (#ancestor-attribute) to determine the sequence of Versions. For Versions
+  that have no `ancestor` value set, compatibility cannot be enforced.
```

Comment:
I think ancestor=null (or absent) is no longer possible. 
So even an Resource with one Version should probably point to itself, for consistency.

Reply:
Agreed, I missed this. I also added a sentence to the ancestor attribute:
>  When creating a version without explicitly setting the `ancestor`
  attribute, the server MUST set the `ancestor` to the most recent Version's
  `versionid` attribute, based on the `createdat` attribute. If multiple
  exist, the first one will be used in alphabetical order. **If no Versions 
  exist, the `ancestor` attribute MUST be set to the `versionid` of the 
  Version being created.**